OpenSim/Common/Component.cpp

```diff
@@ -1531,7 +1531,8 @@ void Component::printInputInfo() const {
                   << std::string(maxlenInputName -
                                  input->getName().length(), ' ')
                   << input->getName() << " : ";
-        if (input->getNumConnectees() == 0) {
+        if (input->getNumConnectees() == 0 || 
+            (input->getNumConnectees() == 1 && !input.isConnected())) {
```

Comment:
This logic assumes that `input->getConnecteeName(i)` is nonempty for all `i` if `input->getNumConnectees() > 1`. Is this assumption correct?